### DISCUSSION TITLE
About yourself

### DISCUSSION THREAD

**Discussion by IvanGao01** [cnosdb/cnosdb#266] [General]
Join us and who want to join us,  You can introduce yourself here and post your thoughts on CnosDB and what you would like to participate in.

**Comment by yanyun827** [comment]
My name is Bart Liu,use C++ five years,Golang five years.I am a new DB developer.I thank CnosDB will be a great TSDB. So I come in.

**Comment by zipper-meng** [comment]
hi mr.01, this is meng. i'm looking for more efficient codes about storage.

**Comment by 0Carrie0** [comment]
My name is Carrie zhang。I‘m working at  CnosDB。

**Comment by ActivePeter** [comment]
请问招实习吗，我目前研0，在东华大学，对rust和基础软件比较感兴趣，希望能找到一些可以实践的机会

**Comment by lutengda** [comment]
My name is Lu Tengda. I have more than two years of experience in database kernel development and five years of experience in C++ development. I am very interested in database development and hope to deepen my career in this field for a long time.Now, I work at CnosDB

**Comment by xuegexi** [comment]
I am Chris Li. I'm working at CnosDB now. Hope we can get better.

**Comment by baiqiubai** [comment]
My name is Liu kai.I‘m working at CnosDB。

**Comment by roseboy-liu** [comment]
My name is liuyongtao. I'm working at CnosDB now.

**Comment by Lunmer** [comment]
My name is Caroline Sun.It's my great honor to join CnosDB.

**Comment by Kree0** [comment]
My name is Yao Kai,  now I'm working at CnosDB.

**Comment by Benxiaohai001** [comment]
My name is Baker X, I'm working as CnnosDB now.

**Comment by liikeuforever** [comment]
My name is Xu Zihang, now I'm working at CnosDB.
